src/demos/wavetable-synth/index.html

```diff
@@ -955,15 +955,29 @@
 }
 
 function start() {
-    // first, get rid of loading animation
     var loading = document.getElementById("loading");
-    loading.innerHTML = "";
 
-    startTime = context.currentTime + 0.160;
-    
     waveTable = loader.getTable(defaultWaveTableName);
     waveTable2 = loader.getTable(defaultWaveTableName);
-    sequence.schedule();    
+
+    const startAudioPlayback = () => {
+        loading.innerHTML = "";
+        startTime = context.currentTime + 0.160;
+        sequence.schedule();
+    };
+
+    if (context.state === 'running') {
+        startAudioPlayback();
+    } else if (context.state === 'suspended') {
+        loading.innerHTML = "Click anywhere or press a key to start playing.";
+        const interactionHandler = () => {
+            document.removeEventListener('mousedown', interactionHandler);
+            document.removeEventListener('keydown', interactionHandler);
+            context.resume().then(() => startAudioPlayback());
+        };
+        document.addEventListener('mousedown', interactionHandler);
```

Comment:
Instead, you can do this:
```js
document.addEventListener('mousedown', interactionHandler, {once: true});
```

Reply:
It's one less line overall to check the state in `interactionHandler`, do you think that would be cleaner?  I don't have a strong preference.

Reply:
I'll land as-is and we can modify later if necessary.  Thank you for the review!